### PR TITLE
update user profile page

### PR DESCRIPTION
**Pull request by sepehr9877**
#102

### CODE REVIEW

File: src/main/resources/templates/profile_candidate.html [hunk 1]
```diff
@@ -200,9 +252,9 @@ <h3>Education</h3>
 
 	<form method="post" action="/profile/saveskill" th:object="${skill}" th:if="${updateskill == true} and ${session.username == candidate.username}">
 
-		<label>Skill</label>
+		<label>Title</label>
```
Comment (MajorTreble) [inline src/main/resources/templates/profile_candidate.html:255]: I prefer Skill and Description

File: src/main/resources/templates/profile_candidate.html [hunk 2]
```diff
@@ -140,10 +181,12 @@ <h3>Education</h3>
 			</form></td>
 
 			</tr>
+			</tbody>
+			
 		</table>
 
 		<form method="post" th:action="@{/profile/updateexperience}" th:object="${newexperience}"  th:if="${session.username == candidate.username}
-				and ${updateeducation == false} and ${updateskill == false}">
+				and ${updateeducation == false} and ${updateexperience == false}">
```
Comment (MajorTreble) [inline src/main/resources/templates/profile_candidate.html:189]: This is incorrect

File: src/main/resources/templates/profile_candidate.html [hunk 3]
```diff
@@ -91,10 +123,12 @@ <h3>Education</h3>
 			</form></td>
 
 			</tr>
+			</tbody>
+			
 		</table>
 
 		<form method="post" th:action="@{/profile/updateeducation}" th:object="${neweducation}" th:if="${session.username == candidate.username} 
-		and ${updateexperience == false} and ${updateskill == false}">
+		and ${updateeducation == false} and ${updateexperience == false}">
```
Comment (MajorTreble) [inline src/main/resources/templates/profile_candidate.html:131]: incorrect

File: src/main/resources/templates/profile_candidate.html (hunk 1 above)
Comment (sepehr9877) [reply src/main/resources/templates/profile_candidate.html:255]: check it , you will see it is a bit confusing , when you put software developer as description and name as java,c++ it doesn't make sense:))). So i consider software developer as title and skills as java,c++

File: src/main/resources/templates/profile_candidate.html (hunk 2 above)
Comment (sepehr9877) [reply src/main/resources/templates/profile_candidate.html:189]: it is not showing the add button if we dont put these conditions

File: src/main/resources/templates/profile_candidate.html (hunk 3 above)
Comment (sepehr9877) [reply src/main/resources/templates/profile_candidate.html:131]: @MajorTreble  check it please if you think it wasnt working very just change the condition , but overall the css and style is working very well , these conditions that i modified are for showing Add button

**Review by nioushamahjoub — APPROVED** [review]

**Review by nioushamahjoub — APPROVED** [review]
it is better to use same css style

**Review by nioushamahjoub — COMMENTED** [review]
it is better to use same css style

File: src/main/resources/templates/profile_candidate.html (hunk 1 above)
Comment (MajorTreble) [reply src/main/resources/templates/profile_candidate.html:255]: If the skill is Java, why would you put software developer as description for that?

This isn't your job title, it is other skills you have picked up

File: src/main/resources/templates/profile_candidate.html (hunk 2 above)
Comment (MajorTreble) [reply src/main/resources/templates/profile_candidate.html:189]: There might be a bug but this is not the fix. The Add button is working for me now.

**Review by MajorTreble — APPROVED** [review]

**Pull request merged**